### ISSUE TITLE
GTK: multiple tabs

### ISSUE THREAD

**Issue by AvtechScientific** [manisandro/gImageReader#521]
It would be nice to have multiple tabs in the GTK OutputEditor. This way you can output different parts of recognized text to different tabs, play with settings to achieve better results and then compare the output in different tabs, etc..

Tabs can be horizontal, like in Gedit and [GtkNotebook widget](https://developer.gnome.org/gtk3/stable/GtkNotebook.html) can be used:

![Screenshot from 2021-05-24 18-49-01](https://user-images.githubusercontent.com/12374315/119395908-d4908480-bcdc-11eb-8c67-a33c1a4605b5.png)

**Comment by manisandro** [comment]
TBH for simple text output, can't you just copy-paste the text to a text editor which supports tabs, and compare the results there?

**Comment by AvtechScientific** [comment]
Comparison is only one use case. I just need a way to work with multiple text files in parallel (e.g. when extracting two/more separate texts located on the same page, like in a newspaper). Those who are not interested in opening several tabs will continue to use the UI as it was before. All that will change for them is that "+" button which opens a new tab (you can see it on the screenshot above, close to the "Open" button in Gedit).

Will you accept a PR if I implement it?

**Comment by manisandro** [comment]
Feature has since landed.
